docs/asciidoc/documentation/globalConfiguration.adoc

```diff
@@ -45,9 +45,34 @@ For example, the following mapping does not allow wildcards:
 </mapping>
 ----
 
+Per default, wildcard mappings are case sensitive. To change this, you
+can enable the wildcard case insensitive policy, either globally or
+at the mapping level:
+
+[source,xml,prettyprint]
+----
+<mapping wildcard-case-insensitive="true">
```

Comment:
I would suggest to demonstrate the relevant part of `<configuration>` block here, as this page is about the global config.

Reply:
Actually the entire page suffers from this problem: https://dozermapper.github.io/gitbook/documentation/globalConfiguration.html

Shall I rewrite the other sections as well?

Reply:
I would appreciate that!